README.md

```diff
@@ -170,6 +170,7 @@ _Related: [Internet of Things (IoT)](#internet-of-things-iot)_
 - [Leon](https://getleon.ai) - Open-source personal assistant who can live on your server. ([Source Code](https://github.com/leon-ai/leon)) `MIT` `Nodejs`
 - [Lidarr](https://lidarr.audio/) - Lidarr is a music collection manager for Usenet and BitTorrent users. ([Source Code](https://github.com/Lidarr/Lidarr)) `GPL-3.0` `C#`
 - [Medusa](https://pymedusa.com/) - Automatic Video Library Manager for TV Shows. It watches for new episodes of your favorite shows, and when they are posted it does its magic. ([Source Code](https://github.com/pymedusa/Medusa)) `GPL-3.0` `Python`
+- [MetaTube](https://github.com/JVT038/MetaTube) - A Web GUI to automatically download music from YouTube add metadata from Spotify, Deezer or Musicbrainz. ([Source Code](https://github.com/JVT038/MetaTube)) `GPL-3.0` `Python / JavaScript`
```

Comment:
- don't add the source code link twice
- Additions that depend on proprietary services outside the user's control must be marked ⚠.
- language/platform should be `Python` (JS is not used server-side)

Reply:
I think I fixed it, is it ok now?